packages/react/src/Tabs/components/Title.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { getComponentClassName } from "../../agent";
+
+export type TitleComponentProps = {
+  enable?: boolean | null;
+  active: boolean;
+  className?: string;
+  classModifier?: string;
+  id: string;
+  children?: React.ReactNode;
+  onChange: (event: React.MouseEvent<HTMLButtonElement>, index: string) => void;
+};
+
+const Title = ({
+  active,
+  onChange,
+  children,
+  classModifier,
+  className,
+  enable,
+  id,
+}: TitleComponentProps) => {
+  const newClassModifier = (classModifier || "").concat(
+    (enable === false ? " disabled" : "").concat(active ? " active" : ""),
+  );
```

Comment:
On pourrait pas faire plus simple et utiliser classnames ?
```suggestion
  const newClassModifier = classNames(classModifier, {"disabled" : enable === false, "active" : active});
```